src/newrelic/cli/args.py

```diff
@@ -90,7 +100,8 @@ def parse_scripted_browser_args(
     parser.add_argument(
         "--script-content",
         type=str,
-        default=args.script_content,
+        action='append',
+        required=True,
```

Comment:
Is this required? IMHO, if we are updating a monitor to disable state, we don't need a script content.